From 00704d4eb30dd12d6f4fcf97320c354ed3eacfa7 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sat, 22 Sep 2012 05:50:38 -0400
Subject: [PATCH] Fix automake-1.12 autoreconf issues

Need -Wno-portablity for '%'-style pattern rules in doc/Makefile.am and
':='-style assignments and $(wildcard *~) in src/hooks/Makefile.am.

Need AM_PROG_CC_C_O for src/applet/applet.c

gettext-0.18.1.1's AM_GNU_GETTEXT requires AM_PROG_MKDIR_P, causing
problems for automake-1.12.
---
 configure.ac           | 8 +++-----
 src/daemon/Makefile.am | 2 +-
 2 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index a78d9f8..42acccf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3,7 +3,7 @@ AC_INIT([abrt],
 	[crash-catcher@fedorahosted.org])
 
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_INIT_AUTOMAKE([-Wall -Wno-portability -Werror foreign])
 # Support silent build rules. Disable by either passing --disable-silent-rules
 # to ./configure or passing V=1 to make
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES])
@@ -12,8 +12,9 @@ AM_SILENT_RULES([yes])
 
 
 AC_DISABLE_STATIC
+AM_PROG_AR
 AC_PROG_LIBTOOL
-AC_PROG_CC
+AM_PROG_CC_C_O
 AC_PROG_LN_S
 
 AC_SYS_LARGEFILE
@@ -30,9 +31,6 @@ GETTEXT_PACKAGE=abrt
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Package name for gettext])
 
-AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.17])
-
 IT_PROG_INTLTOOL([0.35.0])
 
 dnl ****** END ****************************************
diff --git a/src/daemon/Makefile.am b/src/daemon/Makefile.am
index 777dfc4..5a5f0ea 100644
--- a/src/daemon/Makefile.am
+++ b/src/daemon/Makefile.am
@@ -63,4 +63,4 @@ EXTRA_DIST = abrt-handle-upload
 DEFS = -DLOCALEDIR=\"$(localedir)\" @DEFS@
 
 install-data-local:
-	$(mkdir_p) '$(DESTDIR)/$(VAR_RUN)'
+	$(MKDIR_P) '$(DESTDIR)/$(VAR_RUN)'
-- 
1.7.12

